File: Anul II/Medii de proiectare si programare/Laborator/lab07/Ubb.BikeContest.Networking/Request/GetParticipantsByTeamRequest.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Ubb.BikeContest.Networking.Request
{
    [Serializable]
    public class GetParticipantsByTeamRequest : IRequest
    {
        private readonly long teamId;

        public GetParticipantsByTeamRequest(long teamId)
        {
            this.teamId = teamId;
        }

        public long TeamId { get {  return this.teamId; } }
    }
}

File: Anul II/Medii de proiectare si programare/Laborator/lab11/java/networking/src/main/java/com/ubb/protocol/request/GetParticipantsByTeamRequest.java
package com.ubb.protocol.request;

public class GetParticipantsByTeamRequest implements Request {
    private final Long teamID;

    public GetParticipantsByTeamRequest(Long teamID) {
        this.teamID = teamID;
    }

    public Long getTeamID() {
        return teamID;
    }
}
